tests/sampletester/ProcessHelper.cs

```diff
@@ -102,6 +103,110 @@ public static void BuildSolution (string solution, string platform, string confi
 		sb.Add (solution);
 		if (!string.IsNullOrEmpty (target))
 			sb.Add ($"/t:{target}");
-		AssertRunProcess ("msbuild", sb.ToArray (), TimeSpan.FromMinutes (5), Configuration.SampleRootDirectory, environment_variables, "build");
+
+		var watch = Stopwatch.StartNew ();
+		var failed = false;
+		try {
+			AssertRunProcess ("msbuild", sb.ToArray (), TimeSpan.FromMinutes (5), Configuration.SampleRootDirectory, environment_variables, "build");
```

Comment:
Adding
```csharp
environment_variables ["MTOUCH_ENV_OPTIONS"] = "--timestamp --timestamp --timestamp --timestamp";
```
would produce more data, from `mtouch` itself.

Reply:
I'll add the timestamp data as well.

Although it ends up confusing/incorrect when building app extensions, since those are built in parallel with the main app: https://gist.github.com/rolfbjarne/570cb3301d35173467a17826f559feb9